metadata/ch.pn.json

```diff
@@ -4,11 +4,6 @@
 		"ns1.idnscan.net",
 		"ns2.idnscan.net"
 	],
-	"wildcards": [
-		"188.40.117.12",
-		"188.40.70.27",
-		"188.40.70.29"
-	],
```

Comment:
This zone appears to still have wildcard behavior.

Reply:
I’ll take a look. The wildcard detection could use the same mechanism we use for whois.